### ISSUE TITLE
Missing captain\README.md

### ISSUE THREAD

**Issue by AmberKiso** [iron-fish/ironfish#965]
### What happened?

Ironfish/ironfish/README.md references the "captain" twice:

> Ironfish SDK wraps all of the generic components of [Captain](./src/captain/README.md) into a project that is specific to Ironfish.

But that file and the directory do not exist in the repo.

### Version

0.1.21

**Comment by asharma13524** [comment]
@AmberKiso was going to jump on this one but it's not clear to me what file we are trying to reference here?

**Comment by AmberKiso** [comment]
@asharma13524 The readme file is here: https://github.com/iron-fish/ironfish/tree/master/ironfish#readme
The first two lines each reference a missing file: ironfish/src/captain/README.md

I'm not sure it's needed, since the directory doesn't exist in this repo.

**Comment by asharma13524** [comment]
@AmberKiso Right, I understood what file. I was just confused about where the "Captain" hyperlink was intended to go. Would you know the right person to reach out to about this? Just looking to hit the ground running and slowly contribute more meaningful here, thanks.

**Comment by AmberKiso** [comment]
You could ask Jason in the #questions discord.
